conf/README.md

```diff
@@ -0,0 +1,17 @@
+Configuring your *-site.xml files
+======================
+
+Please look at [a full list of Hadoop settings](http://hadoop.apache.org/docs/r2.6.0/hadoop-project-dist/hadoop-hdfs/hdfs-default.xml)
```

Comment:
We should point to the docs that match the version of HDFS we actually use/install.
Of course, I'd rather we update to 2.6.0 than link to 2.3.0/2.4.0.